src/index.js

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2000-2017 TeamDev Ltd. All rights reserved.
+ * TeamDev PROPRIETARY and CONFIDENTIAL.
+ * Use is subject to license terms.
+ */
+
+const {
+  Analyzer, FSUrlLoader, PackageUrlResolver, generateAnalysis
+} = require('polymer-analyzer');
+const escape = require('html-escape');
+const fs = require('fs');
+const path = require('path');
+
+module.exports = {
+  generate: argv => {
+    const templateFilePath = path.join(__dirname, "template", "element-doc-template.html");
+    const itemsRootPath = path.resolve(argv[2] || './');
+
+    const analyzer = new Analyzer({
+      urlLoader   : new FSUrlLoader(itemsRootPath),
+      urlResolver : new PackageUrlResolver()
+    });
+
+    analyzer.analyzePackage().then(_package => {
+      const metadata = generateAnalysis(_package, '');
+      let elements;
+
+      const processingPackage = !Boolean(
+          metadata.elements.find(item => item.path.indexOf(path.sep) === -1));
+
+      elements = splitByElements(metadata.elements, itemsRootPath);
```

Comment:
It seems that elements is initialized only once. Consider making it a const variable.